server/src/domain/media/media.service.ts

```diff
@@ -87,6 +87,18 @@ export class MediaService {
     return true;
   }
 
+  async handleGenerateThumbhashThumbnail({ id }: IEntityJob): Promise<boolean> {
+    const [asset] = await this.assetRepository.getByIds([id]);
+    if (!asset || !asset.resizePath) {
```

Comment:
nit:
```suggestion
    if (!asset?.resizePath) {
```

Reply:
This is how all the other jobs are right now. You could make a separate PR to go through and fix all of them :stuck_out_tongue:

Reply:
Got to start somewhere I guess